css/DesktopVersion.css

```diff
@@ -0,0 +1,417 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (min-width: 768px),
+  (min-width: 1024),
+  (min-width: 1440px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #dfe1e6;
+  }
+
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: white;
+    color: black;
+    padding: 10px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-left: 90px;
+  }
+
+  nav ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    justify-content: flex-end; /* new */
+  }
+
+  nav ul li {
+    display: inline-block;
+    margin-right: 20px;
+  }
+
+  nav ul .nameJacK {
+    position: absolute;
+    top: 5px;
+    left: 0;
+    font-weight: bolder;
+    font-size: 20px;
+    margin-left: 10px;
+    color: #6070ff;
+  }
+
+  nav ul li a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  main {
+    display: grid;
+    grid-template-rows: repeat(1fr);
+    justify-content: center;
+    align-items: center;
+    height: 900px;
+    background: white;
+    background-image: url(../DeskImg/Header\ bg@2x.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-bottom-left-radius: 200px;
+  }
+
+  .titleText .beConnect {
+    width: 217px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .titleText h1 {
+    width: 920px;
+    height: 122px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-top: 30px;
+  }
+
+  .titleText p {
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 920px;
+    height: 98px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .LinkNetwork ul li {
+    display: flex;
+    justify-content: flex-start;
+    position: relative;
+    left: -59px;
+    margin: 10px;
+  }
+
+  .LinkNetwork ul {
+    display: flex;
+    justify-content: flex-start;
+    list-style: none;
+    flex-wrap: nowrap;
+    margin: 0;
+
+    /* new */
+  }
+
+  /* none displayer session */
+  .imgOff {
+    display: none;
+  }
+
+  /* Trying design */
+  .template {
+    display: grid;
+    justify-content: center;
+    max-width: 80%;
+    margin: 0 auto;
+    grid-template-columns: repeat(2, 1fr);
+    background-color: #fff;
+    border-radius: 24px;
+    gap: 20px;
+    margin-top: 100px;
+    padding: 20px;
+  }
```

Comment:
- [ ] Good work building your Desktop Works section 💯. However, please kindly ensure to fix the margins in your design to stick as much as possible to the Figma design as requested by the previous reviewer. 🤓

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/236356638-814727b0-5b7e-4a38-8642-d787edcfccfb.png)

This is your design 👇🏽

![design](https://user-images.githubusercontent.com/116701485/236356646-907e9477-0b7e-44de-9aaf-7fffa47c3ec9.png)